.eslintrc.js

```diff
@@ -64,7 +64,6 @@ module.exports = {
     ],
 
     // TODO - enable these new recommended rules
```

Comment:
I dug into why the build is showing so many failures for `eslint-utils`. I think it's that `packages/experimental-utils/typings/eslint-utils.d.ts` is not included in the program's list of files, so the `declare module` isn't picked up by the program -> type checker.

The cleanest solution IMO would be to add a DefinitelyTyped `@types/` package, which I'll try to do soon :)

Reply:
I don't think that's right because the `.d.ts` file should be in the program via this:
https://github.com/typescript-eslint/typescript-eslint/blob/dac8845c6821d51cea5bafdc0be2e9a399cea68c/packages/experimental-utils/tsconfig.json#L7

Reply:
I dug into this in https://www.twitch.tv/videos/1049146703 but don't see any existing rule disables on master that still would necessitate this... to save us both the trouble of watching a 30 minute stream I'll just resolve this for now!